README-ocp4.md

```diff
@@ -71,14 +70,14 @@ cd federation-dev/
 <a id="markdown-create-the-two-openshift-clusters" name="create-the-two-openshift-clusters"></a>
 ## Create the two OpenShift clusters
 
-Follow the [developer preview instructions](https://cloud.openshift.com/clusters/install) for installing two OpenShift 4.0 clusters on AWS.
+Follow the [developer preview instructions](https://cloud.redhat.com/openshift/install) for installing two OpenShift 4.0 clusters on AWS.
```

Comment:
Possibly change this to 4.1 as that will be the GA release version

Reply:
Done.